io.github.rfrench3.scopebuddy-gui.yml

```diff
@@ -0,0 +1,38 @@
+id: io.github.rfrench3.scopebuddy-gui
+runtime: org.kde.Platform
+runtime-version: '6.9'
+sdk: org.kde.Sdk
+base: io.qt.PySide.BaseApp
+base-version: '6.9'
+command: scopebuddygui
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=xdg-config/scopebuddy:create
```

Comment:
this would need an [exception](https://docs.flathub.org/docs/for-app-authors/linter#exceptions)